typescript-getting-started/README.md

```diff
@@ -0,0 +1,143 @@
+## Cloud Functions for Firebase: getting started with TypeScript
+
+This is an example Firebase project for using TypeScript with
+[Cloud Functions for Firebase](https://firebase.google.com/products/functions)
+
+Pre-requistes:
```

Comment:
This is a little odd to have both a fully functional template project and then instructions that don't use them. Maybe the steps for manual setup can be in DIY.md and linked from main README.md?

Reply:
good idea